permissions/Compatibility.permission

```diff
@@ -0,0 +1,41 @@
+# -*- mode: sh -*-
+# This provides compatibility for older apps
+# The permission is not shown to user,
+# thus it ***must not*** give access to additional data or hardware resources
+
+# x-sailjail-translation-catalog =
+# x-sailjail-translation-key-description = permission-la-compatibility
+# x-sailjail-description = Compatibility
+# x-sailjail-translation-key-long-description = permission-la-compatibility_description
+# x-sailjail-long-description =
+
+# PDF tools
+private-bin pdfattach
```

Comment:
pdfattach is perhaps the one that could have the most nefarious use cases, but nothing that couldn't be replicated.